src/lava/proc/conv/utils.py

```diff
@@ -482,7 +482,10 @@ def dst_id(x: int, y: int, c: int):
                     dsts.append(dd.flatten())
                     wgts.append(weight.flatten())
 
-    srcs = np.concatenate(srcs).astype(int)
-    dsts = np.concatenate(dsts).astype(int)
-    wgts = np.concatenate(wgts)
+    if len(srcs) >= 1:
+        srcs = np.concatenate(srcs).astype(int)
+    if len(dsts) >= 1:
+        dsts = np.concatenate(dsts).astype(int)
+    if len(wgts) >= 1:
+        wgts = np.concatenate(wgts)
```

Comment:
Fix for test_conv_to_sparse failure. Not my code or unit test. @bamsumit  is this your code/test and if so does this fix look problematic at all?

```
======================================================================
ERROR: test_conv_to_sparse (tests.lava.proc.conv.test_utils.TestConv)
Tests translation of a conv to a sparse layer
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/lava/lava/tests/lava/proc/conv/test_utils.py", line 138, in test_conv_to_sparse
    dst, src, wgt = utils.conv_to_sparse(input_shape=input.shape,
  File "/Users/runner/work/lava/lava/src/lava/proc/conv/utils.py", line 488, in conv_to_sparse
    dsts = np.concatenate(dsts).astype(int)
  File "<__array_function__ internals>", line 200, in concatenate
ValueError: need at least one array to concatenate

----------------------------------------------------------------------
Ran 566 tests in 190.079s

FAILED (errors=1, skipped=18)
Error: Process completed with exit code 1.
```

Reply:
The issue is resolved by Sumit now